packages/notebook/src/widget.ts

```diff
@@ -852,6 +853,10 @@ class Notebook extends StaticNotebook {
     if (newValue === oldValue) {
       return;
     }
+    // Ensure that the previous cell's editor is blurred.
```

Comment:
Question: what happens if the active cell changed by the user clicking somewhere else with the mouse. In that case, hasn't the blur already happened? At some point, in the classic notebook, we had weird blur/focus cycles because of the differences in keyboard and mouse interactions.